### DISCUSSION TITLE
What are the keybindings for folding the treeview

### DISCUSSION THREAD

**Discussion by tcoopman** [jesseduffield/lazygit#1909] [Q&A] [answered]
Looking at [this](https://github.com/jesseduffield/lazygit/issues/1227#issuecomment-815722421) comment it seems that the files treeview has folding, but I can't find the keybinding anywhere?

**Comment by cynicaljoy** [comment, marked as answer]
You can use [enter](https://github.com/jesseduffield/lazygit/blob/8b103b16bd3a06d0e661e507832459fae7ec1198/docs/keybindings/Keybindings_en.md?plain=1#L130)
